### ISSUE TITLE
The "comment" context key doc is wrong

### ISSUE THREAD

**Issue by alainxi** [ynput/ayon-documentation#87]
On this page :   https://ayon.ynput.io/docs/admin_settings_project_anatomy/
It is stated that "comment" key in an Anatomy template means this : "Publishing commit message".
But in fact, it is not related to publish, and its name is misleading (comment vs subversion).

I have tested this :

If that key is used in a publish template, it has no effect (I entered some text in the "Comment" field of both the DCC publish gui and its TrayPublisher counterpart, without any effect on the published file name).

If that key is used in a worfile template, it adds to the file name the content of the "Subversion" field that is located in this menu from the DCC (Blender in my tests) : "Ayon-->Work Files-->Save As" 

To sum up :
- The "Comment" field in Publish GUI is **not** transmitted to the "comment" key inside publish Anatomy templates.
- The "Subversion" field in Workfiles GUI **is** transmitted to the "comment" key inside work Anatomy templates.

**Comment by alainxi** [comment]
Thanks for having fixed this !
